src/HMRC.ESFA.Levy.Api.Client/SignificantProcessingDates.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace HMRC.ESFA.Levy.Api.Client
+{
+    internal class SignificantProcessingDates
+    {
+        public DateTime DateProcessorInvoked { get; set; }
```

Comment:
the DateProcessorInvoked is a runtime value and the rest are configuration so you should probably break these out